smartcontractkit/external-adapters-js
# validate empty string parameters

## Closes 19390

## Description

Validates empty string adapter request parameters.

## Changes

 - Update the validator to validate empty string request parameters
 - Add unit tests to verify that the validator validates empty string request parameters

## Steps to Test

1. Run any adapter
2. Pass a request with one of the required fields as an empty string
3. Verify that the adapter returns a 400 error response

## Quality Assurance
